editor/src/app/inputs/text-input.service.ts

```diff
@@ -44,16 +44,56 @@ export class TextInputService {
     if (!event.target.value && !this.hideIcon) {
       this.showIcon.next(true);
     }
-    this.updateField(event);
+    this.updateField(event, null);
   }
 
-  updateField(event) {
+  updateField(event, label) {
+
     if (event instanceof KeyboardEvent && (event.key === 'Escape' || event.keyCode === 27)) {
       (event.target as HTMLInputElement).value = this.lastValue;
       (event.target as HTMLInputElement).blur();
       return null;
     }
 
+    if (event.key === 'ArrowDown' || event.keyCode === 40 || event.key === 'ArrowUp' || event.keyCode === 38) {
+      const myLabel = ['Page margins', 'Margins', 'Gallery margins', 'Entry margins'];
+      const bigUnits = ['px', '%', 'pt'];
+      const litleUnits = ['em', 'rem', 'vw', 'vh'];
+
+      if (!myLabel.includes(label)) {
```

Comment:
This is a very weird implementation. If you wish to filter out specific settings that will have this feature you should do it before - when the inputs are created, you can add a parameter, that determines if they will have this arrow key feature.

That should most likely be determined by the type of the setting input.